Content.Client/Administration/UI/BwoinkWindow.xaml

```diff
@@ -5,15 +5,20 @@
     HeaderClass="windowHeaderAlert"
     TitleClass="windowTitleAlert"
     Title="{Loc 'bwoink-user-title'}" >
-    <SplitContainer Orientation="Horizontal">
-        <cc:PlayerListControl Access="Public" Name="ChannelSelector" HorizontalExpand="True" SizeFlagsStretchRatio="1" />
+    <SplitContainer Name="Split" Orientation="Horizontal">
+        <BoxContainer Orientation="Horizontal">
+            <cc:PlayerListControl Access="Public" Name="ChannelSelector" HorizontalExpand="True" SizeFlagsStretchRatio="1" />
+        </BoxContainer>
```

Comment:
Not sure why this is needed, but the `SplitContainer` doesn't register the `PlayerListControl` size changes unless it's there.